rules/auth0_rules/auth0_user_invitation_created.py

```diff
@@ -0,0 +1,43 @@
+import re
+
+from global_filter_auth0 import filter_include_event
+from panther_auth0_helpers import auth0_alert_context, is_auth0_config_event
+from panther_base_helpers import deep_get
+
+org_re = re.compile(r"^/api/v2/organizations/[^/\s]+/invitations$")
+
+
+def rule(event):
+    if not filter_include_event(event) or not is_auth0_config_event(event):
+        return False
+
+    return invitation_type(event) is not None
+
+
+def title(event):
+    inv_type = invitation_type(event)
+    if inv_type == "tenant":
+        invitee = deep_get(event, "data", "details", "request", "body", "owners")[0]
```

Comment:
I think the suggestion will also have an issue since the 0th element of [] will also be an `IndexError`. I'll update this chunk in general for the zero-length case.

Reply:
oooh, yes, you are correct! thank you!

Reply:
Updated with `try` block